pkg/driver/node.go

```diff
@@ -419,6 +474,28 @@ func parseVolumeId(volumeId string) (fsid, subpath, apid string, err error) {
 	return
 }
 
+func (p podAuth) setPodCredentials() (e error) {
+
+	response, err := k8sv1.CoreV1Interface.ServiceAccounts(p.k8sClient, p.nameSpace).Get(p.ctx, p.serviceAccount, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Unable to get the service account description %s", err)
+		return err
+	}
+	roleArn := response.Annotations["eks.amazonaws.com/role-arn"]
+
+	errWrite := os.WriteFile("/home/token", []byte(p.jwtToken), 0644)
```

Comment:
Is it worth having the token location be configurable? (Genuine question I'd be interested in your views)